elements/lisk-tree/src/sparse_merkle_tree/branch.ts

```diff
@@ -12,35 +12,43 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { Leaf } from './leaf';
-import { branchHash } from './utils';
+import { hash } from '@liskhq/lisk-cryptography';
+import { branchDataBuffer } from './utils';
 import { NodeSide } from './constants';
 
 export class Branch {
-	private _left: Leaf;
-	private _right: Leaf;
+	private _leftHash: Buffer;
+	private _rightHash: Buffer;
 	private _hash: Buffer;
```

Comment:
I think this is no longer needed as a separate class attribute since it can always be computed from the buffer (data/digest). 
Same for `Leaf` class

Reply:
yeah I agree but still it is handy to use as a property instead of using hash function within update/delete funcs. but not any strong opinion on this